tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
The `tsconfig` file now supports globbing, this means you can use the new `include` options instead of `filesGlob` and remove the `files` array completely.